woocommerce/woocommerce-ios
# [Tablet Orders] Update selected state design in Order List

Closes: #11608 

## Description
This PR updates the `OrderTableViewCell` style when selected following the designs on 6kkCwI9VOEYcidp6UT5bln-fi-192_25891

| Before | After |
|--------|--------|
| <img width="289" alt="Screenshot 2024-01-04 at 08 09 32" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/d91a697c-c8b9-4233-a899-e5c174ea8609"> | <img width="312" alt="Screenshot 2024-01-04 at 08 08 35" src="https://github.com/woocommerce/woocommerce-ios/assets/3812076/475aad37-3ce6-4d34-bb13-0c8172ae7118"> | 

## Changes
- Updates the current `.clear` cell background to be `.wooCommercePurple(.shade0)` instead
- When configuring the cell, we set `focusEffect` to `nil`.  This is a new property since iOS15 that was applying a default focus style to the cell which was adding a border around it.
```
    /// Describes a visual effect to apply when this item is focused. If this property is nil no effect will be applied when this view becomes focused.
    @available(iOS 15.0, *)
    @NSCopying open var focusEffect: UIFocusEffect?
```

## Testing instructions
1. Set the `.splitViewInOrdersTab` feature flag to `true`
2. Run the app on an iPad simulator
3. Navigate to Orders. Observe how the updated style is applied to the pre-selected (if possible) order, and subsequently applies to any other selected order.

![Simulator Screen Recording - iPad Pro (12 9-inch) (6th generation) - 2024-01-05 at 14 02 24](https://github.com/woocommerce/woocommerce-ios/assets/3812076/ca9ee248-6daf-437b-80ed-39999cca3410)
